refarch-eai/src/main/resources/application-local.yml

```
@@ -1,2 +1,5 @@
 server:
-  port: 8085
\ No newline at end of file
+  port: 8085
+
+logging:
+  include-application-group: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)